apps/api/src/app/routes/__chainId/markets/__baseTokenAddress-__quoteTokenAddress/slippageTolerance.ts

```diff
@@ -72,18 +89,63 @@ const root: FastifyPluginAsync = async (fastify): Promise<void> => {
     },
     async function (request, reply) {
       const { chainId, baseTokenAddress, quoteTokenAddress } = request.params;
+
       fastify.log.info(
-        `Get default slippage for market ${baseTokenAddress}-${quoteTokenAddress} on chain ${chainId}`
+        `Get default slippage for market ${baseTokenAddress}-${quoteTokenAddress} on chain ${chainId}. Query: %s'`,
+        JSON.stringify(request.query)
       );
-      const slippageBps = await slippageService.getSlippageBps(
+      const slippageBps = await slippageService.getSlippageBps({
+        chainId,
         baseTokenAddress,
+        quoteTokenAddress,
+      });
+      reply.header(
+        CACHE_CONTROL_HEADER,
+        getCacheControlHeaderValue(CACHE_SECONDS)
+      );
+      reply.send({ slippageBps });
+    }
+  );
+
+  fastify.get<{
+    Params: RouteSchema;
+    Reply: {
+      baseToken: VolatilityDetails | null;
+      quoteToken: VolatilityDetails | null;
+    };
+  }>(
+    '/volatilityDetails',
```

Comment:
Ah thought that just crossed my mind, feel free to ignore.

I'm not very happy with an endpoint just for debugging.
What about having an optional (maybe hidden) parameter in the regular call that returns all of this data with the slippage request?

Reply:
Yes, is a fair point, but also I don't love adding this additional data in the slippage tolerance endpoint, cause this is the one people will use and I don't want to expose these internal details. 

Although I used the volatilityDetails for a market for debugging, I think it doesn't harm to have it there. It returns nice information about a market volatility. 

I can also hide it in swagger to make it "not official" would that help?

Reply:
Leandro, I will merge and address things as follow ups, cause I want to test this in CoW Swap so I want to deploy it to staging. 

This endpoint is not used in CoW Swap, so I hope the resolution of this one won't affect there